config/replacements.php

```diff
@@ -157,6 +157,8 @@
     'Zend\\Expressive' => 'Expressive',
     'Zend\\\\Expressive' => 'Expressive',
     'ZendAuthentication' => 'LaminasAuthentication',
+    'ZendAcl' => 'ZendAcl',
```

Comment:
`ZendAcl` to `ZendAcl`? Is this correct?

Reply:
Argh... of course not ! to LaminasAcl! Thanks, changing now :)